OFEP-single-context-paradigm.md

```diff
@@ -82,3 +84,22 @@ interface Client {
 #### Remove context mutator
 
 The global context has a one-to-one correspondence to the providers cache of evaluated flags. It's unreasonable or impossible to reconcile multiple client-level contexts with this state. The context mutator of the client must be removed.
+
+### Hook changes
+
+#### Remove return value from before stage
+
+Context cannot be modified at evaluation, so `before` stage 
```

Comment:
This change would only apply when in "single-context" mode, right?

Reply:
I wouldn't describe single context as a "mode", since it can't be changed, but yes, this would only apply to SDKs which are designed to be used in single user applications, such as a web SDK.